style.css

```diff
@@ -1678,6 +1678,10 @@ body { background: #b0d4c8; font-family: "Roboto", sans-serif; }
 
 body.brown-bg { background: #FAF7EC; }
 
+.bold { font-weight: bold; }
+.activefilter-tag { background-color: transparent; text-align: left; cursor: pointer; }
+.clearall { cursor: pointer; text-transform: uppercase; }
+
```

Comment:
@ashish173 
Ashish I am tagging you at the lines that you need to see for adding changes in presentation layer.